Clyde-Cli Install is Broken

Couldn't get clyde-cli to work on my NixOS machine, so I tried to install it in a fresh Ubuntu 24.10 container and confirmed there seems to be an issue:

```
node:internal/modules/cjs/loader:599
      throw e;
      ^

Error: Cannot find module '/usr/local/lib/node_modules/@clyde-lang/cli/node_modules/@clyde-lang/parser/dist/cjs/index.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:1182:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:1170:15)
    at resolveExports (node:internal/modules/cjs/loader:592:14)
    at Module._findPath (node:internal/modules/cjs/loader:669:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1131:27)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/@clyde-lang/cli/dist/cjs/parser.js:9:18)
    at Module._compile (node:internal/modules/cjs/loader:1358:14) {
  code: 'MODULE_NOT_FOUND',
  path: '/usr/local/lib/node_modules/@clyde-lang/cli/node_modules/@clyde-lang/parser/package.json'
}

Node.js v20.14.0
```

Tried both `npm install -g @clyde-lang/cli` and `yarn global add @clyde-lang/cli`

--- a/cli/CHANGELOG.md
+++ b/cli/CHANGELOG.md
@@ -2,6 +2,16 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## 3.0.1 (2024-06-20)
+
+### Fixed
+
+- Dist files started being ignored by npm. Explicitly adding them and updating parser and interpreter.
+
+### Thanks
+
+- Thanks to Aaron (@clogwyn55) for reporting this issue.
+
 ## 3.0.0 (2024-02-14)
 
 ### Breaking changes

--- a/cli/package.json
+++ b/cli/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@clyde-lang/cli",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "CLI tool for executing, parsing and debugging Clyde dialogue files",
   "main": "./dist/cjs/index.js",
   "author": "Vinicius Gerevini <viniciusgerevini@gmail.com>",
@@ -14,9 +14,10 @@
   "bin": {
     "clyde": "./dist/cjs/cli.js"
   },
+  "files": ["./dist", "README.md", "CHANGELOG.md"],
   "dependencies": {
-    "@clyde-lang/interpreter": "4.0.0",
-    "@clyde-lang/parser": "2.3.0",
+    "@clyde-lang/interpreter": "4.0.1",
+    "@clyde-lang/parser": "2.3.2",
     "yargs": "^16.2.0"
   },
   "devDependencies": {

--- a/editor/package.json
+++ b/editor/package.json
@@ -4,8 +4,8 @@
   "private": true,
   "homepage": "https://viniciusgerevini.github.io/clyde",
   "dependencies": {
-    "@clyde-lang/interpreter": "4.0.0",
-    "@clyde-lang/parser": "2.3.0",
+    "@clyde-lang/interpreter": "4.0.1",
+    "@clyde-lang/parser": "2.3.2",
     "@fortawesome/fontawesome-svg-core": "^1.2.32",
     "@fortawesome/free-solid-svg-icons": "^5.15.1",
     "@fortawesome/react-fontawesome": "^0.1.14",

--- a/interpreter/CHANGELOG.md
+++ b/interpreter/CHANGELOG.md
@@ -2,6 +2,16 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## 4.0.1 (2024-06-20)
+
+### Fixed
+
+- Dist files started being ignored by npm. Explicitly adding them and updating parser.
+
+### Thanks
+
+- Thanks to Aaron (@clogwyn55) for reporting this issue.
+
 ## 4.0.0 (2024-02-14)
 
 ### Breaking changes

--- a/interpreter/package.json
+++ b/interpreter/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@clyde-lang/interpreter",
-  "version": "4.0.0",
+  "version": "4.0.1",
   "description": "Interpreter for Clyde dialogue language",
   "main": "./dist/cjs/index.js",
   "types": "./dist/mjs/index.d.ts",
@@ -27,11 +27,12 @@
     },
     "./package.json": "./package.json"
   },
+  "files": ["./dist", "README.md", "CHANGELOG.md"],
   "jest": {
     "preset": "ts-jest",
     "testEnvironment": "node"
   },
   "dependencies": {
-    "@clyde-lang/parser": "2.3.0"
+    "@clyde-lang/parser": "2.3.2"
   }
 }

--- a/parser/CHANGELOG.md
+++ b/parser/CHANGELOG.md
@@ -2,6 +2,15 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## 2.3.2 (2024-06-20)
+
+### Fixed
+
+- Dist files started being ignored by npm.
+
+### Thanks
+
+- Thanks to Aaron (@clogwyn55) for reporting this issue.
 
 ## 2.3.0 (2024-02-14)
 

--- a/parser/package.json
+++ b/parser/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@clyde-lang/parser",
-  "version": "2.3.0",
+  "version": "2.3.2",
   "description": "Parser for Clyde dialogue language",
   "main": "./dist/cjs/index.js",
   "types": "./dist/mjs/index.d.ts",
@@ -29,6 +29,7 @@
     "./package.json": "./package.json"
   },
   "dependencies": {},
+  "files": ["./dist", "README.md", "CHANGELOG.md"],
   "jest": {
     "preset": "ts-jest",
     "testEnvironment": "node"

